doc/source/whatsnew/v0.18.2.txt

```diff
@@ -412,6 +412,32 @@ Furthermore:
 - Passing duplicated ``percentiles`` will now raise a ``ValueError``.
 - Bug in ``.describe()`` on a DataFrame with a mixed-dtype column index, which would previously raise a ``TypeError`` (:issue:`13288`)
 
+.. _whatsnew_0182.api.difference:
+
+``Index.difference`` and ``Index.symmetric_difference`` will now, more consistently, treat ``NaN`` values as amy other values.
+
+.. ipython:: python
```

Comment:
add the issue number here at the end of the first sentence. (if no direct issue, then use the PR number)